package.json

```diff
@@ -94,7 +94,6 @@
 		"@wordpress/npm-package-json-lint-config": "file:packages/npm-package-json-lint-config",
 		"@wordpress/postcss-themes": "file:packages/postcss-themes",
 		"@wordpress/scripts": "file:packages/scripts",
-		"babel-loader": "8.0.6",
```

Comment:
This needs to be explicit as long as one dependency has it as a peer dependency, which they do.

Otherwise you are relying on another dependency bringing it in as a dependency, but whenever that dependency is removed, things break.

Reply:
I see it now 👍